src/style.css

```diff
@@ -0,0 +1,111 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img {
+  margin: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: #060b35;
+  font-style: initial;
+  margin: 4px 0 16px 16px;
+}
+
+.card__reviews-star {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+  margin: 0 16px 24px;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: inline-block;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__reviews-star-reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px;
+}
+
+.card__price-text {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+}
+
+.card__price-price {
+  color: #060b35;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+}
+
+.card__bottom {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #00acdc;
```

Comment:
create a variable

Reply:
Not resolved. You need to create CSS variables for all repeatable colors.